agent/bootstrap.go

```diff
@@ -265,9 +265,8 @@ func acquireLock(path string, seconds int) (*lockfile.Lockfile, error) {
 	for true {
 		err := lock.TryLock()
 		if err != nil {
-			if err == lockfile.ErrBusy {
-				// Keey track of how many times we tried to get the
-				// lock
+			if te, ok := err.(interface{ Temporary() bool }); ok && te.Temporary() {
```

Comment:
Oh boo, I hate having to do error inspection like this. Makes for such an annoying API.

So reading through the docs, `Temporary` is the "you should try again" error?